content/en/continuous_integration/troubleshooting.md

```diff
@@ -20,9 +20,44 @@ kind: documentation
 
 If you can see test results data in the **Test Runs** tab, but not the **Tests** tab, Git metadata (repository, commit and/or branch) is probably missing. To confirm this is the case, open a test execution in the [Test Runs][3] section, and check that there is no `git.repository_url`, `git.commit.sha`, or `git.branch`. If these tags are not populated, nothing shows in the [Tests][4] section.
 
-1. Tracers first try to fetch Git metadata using the local `.git` folder by executing `git` commands. This is the preferred approach, as it populates all Git metadata fields, including commit message, author and committer information. Ensure the `.git` folder is present and the `git` binary is installed and in `$PATH`.
-2. If the `.git` folder is not present, or the `git` binary is not installed, tracers fallback to use the environment variables set by the CI provider to collect Git information. See the [Running tests inside a container][6] page for a list of environment variables that the tracer attempts to read for each supported CI provider. At a minimum, this populates the repository, commit hash, and branch information.
-3. If no CI provider environment variables are found, tests results are sent with no Git metadata.
+1. Tracers first try to use the environment variables set by the CI provider to collect Git information. See the [Running tests inside a container][6] page for a list of environment variables that the tracer attempts to read for each supported CI provider. At a minimum, this populates the repository, commit hash, and branch information.
+2. Tracers also try to fetch Git metadata using the local `.git` folder by executing `git` commands. This populates all Git metadata fields, including commit message, author and committer information. Ensure the `.git` folder is present and the `git` binary is installed and in `$PATH`. This information will be used to populate attributes not detected in the previous step.
+3. Git information can also be provided manually using environment variables, which override any information detected by any of the previous steps. The supported environment variables for providing Git information are the following:
+
+`DD_GIT_REPOSITORY_URL`
```

Comment:
We can also include the examples in this page

Reply:
Sure, dont know how I missed them